eng/Packages.props

```diff
@@ -120,6 +123,7 @@
     <PackageReference Update="Swashbuckle.AspNetCore.Annotations" Version="5.2.1" />
     <PackageReference Update="Swashbuckle.AspNetCore.Newtonsoft" Version="5.2.1" />
     <PackageReference Update="Swashbuckle.AspNetCore.SwaggerGen" Version="5.2.1" />
+    <PackageReference Update="System.CommandLine" Version="2.0.0-beta1.21308.1" />
```

Comment:
I would prefer using Mono.Options like the rest of the tools in this repo, rather than add another.

Reply:
darc actually uses something different (CommandLineParser, which is what the CLI used to use). I'd prefer to use this as this is what the official .NET CLI currently uses, and it's especially simple for these cases.